auth/src/main/java/org/aerogear/mobile/auth/authenticator/DefaultAuthenticateOptions.java

```diff
@@ -22,4 +57,12 @@ public Activity getFromActivity() {
     public int getResultCode() {
         return resultCode;
     }
+
+    public boolean getSkipCertificatePinningChecks() {
+        return skipCertificatePinningCheck;
+    }
+
+    public static Builder newBuilder() {
```

Comment:
I'm not sure this method is really useful.
However, there is an inconsistency: this public method returns an instance of the Builder object which is not public.